### PR TITLE
Cart model test

### LINKED ISSUE #73: model tests for cart model

### PR DESCRIPTION
**Pull request by anlewis**
Closes #73 

Description of Changes:
-add cart model spec

@anlewi5, @josimccllelan, @aziobrow

### CODE REVIEW

**Comment by anlewis** [comment]
Hey, @JosiMcClellan and @aziobrow, I feel weird using instance variables in my spec. Can you check that out and let me know the right way to do it? Otherwise, I think everything else should be fine.

**Review by JosieMcClellan — CHANGES_REQUESTED** [review]
I changed some (cosmetic) things in the cart spec, but the styling breaks some feature specs.  Probably just a button renamed button or something, but could you fix that and PR again?

**Comment by anlewis** [comment]
All the tests should be passing now.

File: spec/models/cart_spec.rb [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  before do
+    item1 = create(:item, id: 1, price: 10)
+    item2 = create(:item, id: 2, price: 10)
+    item3 = create(:item, id: 3, price: 10)
+    @cart = Cart.new("1" => 2, "2" => 3, "3" => 4)
+  end
+
+  describe '#contents' do
+    it 'stores contents in a hash' do
+      expect(@cart.contents).to eq({"1" => 2, "2" => 3, "3" => 4})
```
Comment (aziobrow) [inline spec/models/cart_spec.rb:14]: Wondering if this is a robust test--in line 9, you created the cart as a hash, and then in line 14 are checking to see if it is a hash--might be more organic if it was capybara'd out.

File: spec/models/cart_spec.rb [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    end
+  end
+
+  describe '#cart_items' do
+    it 'returns actual item records keyed to quantities' do
+      expect(@cart.cart_items).to eq({Item.first => 2, Item.second => 3, Item.third => 4})
+    end
+  end
+
+  describe '#total_price' do
```
Comment (aziobrow) [inline spec/models/cart_spec.rb:69]: at some point, we want to check to be sure that the price in cents is formatted correctly for the user

**Review by aziobrow — APPROVED** [review]
Really thorough testing!

**Pull request merged**